server/src/tasks.ts

```diff
@@ -41,6 +41,8 @@ export class TasksAndModels extends EventEmitter {
           console.error(e instanceof Error ? e.message : e.toString())
           return
         }
+        // Create the directory if it does not exist
```

Comment:
it's weird we're getting a "dir missing" error, since the directory has a committed `.empty` file in it and thus exists in the repo

I think we want to keep exactly one of the two solutions: either the recursive `mkdir` or the `.empty` file (the latter you can remove)